### ISSUE TITLE
issue building in arch linux

### ISSUE THREAD

**Issue by thejacer87** [FasterMelee/FasterMelee-installer#11]
I've got all the dependencies (i think) but when i try to build with yaourt i get this error: 

```
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/include-fixed/limits.h:168:0,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/include-fixed/syslimits.h:7,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/include-fixed/limits.h:34,
                 from /tmp/yaourt-tmp-jbennest/aur-dolphin-emu-faster-melee/src/dolphin-emu-faster-melee/build/Source/CMakeFiles/pch.dir/pch.h:35,
                 from <command-line>:0:
/tmp/yaourt-tmp-jbennest/aur-dolphin-emu-faster-melee/src/dolphin-emu-faster-melee/Source/Core/VideoBackends/OGL/RasterFont.cpp:18:18: error: expected unqualified-id before numeric constant
 static const int CHAR_WIDTH = 8;
                  ^
make[2]: *** [Source/Core/VideoBackends/OGL/CMakeFiles/videoogl.dir/build.make:264: Source/Core/VideoBackends/OGL/CMakeFiles/videoogl.dir/RasterFont.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1140: Source/Core/VideoBackends/OGL/CMakeFiles/videoogl.dir/all] Error 2
make: *** [Makefile:161: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Makepkg was unable to build dolphin-emu-faster-melee.
```

**Comment by Ptomerty** [comment]
Try again with the latest build, this issue was fixed in #7.

**Comment by thejacer87** [comment]
yep, it worked, the AUR package must be behind. thanks
